ide/coqide/preferences.ml

```diff
@@ -1060,25 +1063,19 @@ let configure ?(apply=(fun () -> ())) parent =
       "External editor"
       ~f:cmd_editor#set
       ~new_allowed: true
-      (predefined@[if List.mem cmd_editor#get predefined then ""
-                   else cmd_editor#get])
+      (predefined@(if List.mem cmd_editor#get predefined then []
+                   else [cmd_editor#get]))
       cmd_editor#get
   in
   let cmd_browse =
-    let predefined = [
-      Coq_config.browser;
-      "netscape -remote \"openURL(%s)\"";
-      "mozilla -remote \"openURL(%s)\"";
-      "firefox -remote \"openURL(%s,new-windows)\" || firefox %s &";
-      "seamonkey -remote \"openURL(%s)\" || seamonkey %s &"
-    ] in
+    let predefined = [Coq_config.browser; {|firefox "%s" &|}; {|seamonkey "%s" &|}; {|chromium "%s" &|}] in
```

Comment:
The two parts of your sentence look contradictory to me… Netscape Navigator was stopped in 2008. These strings are just suggested choices; I didn’t even know that “mozilla” ever was a command, but it and “netscape” are surely now too old/rare to have their place here. It would probably be more sensible to add “safari” or “chrome”; but the default command will usually work. If people had already selected the “netscape” or “mozilla” command, it is written in their preferences file and will be reused even with this change.

I don’t think it is easier to read on separate lines… I like using the full length of lines, to see more code on my screen at once. I also have a tendency not to like other people’s coding styles sometimes, but I think we have to learn that a coding style is okay as long as it is coherent (meaning it follows precisely defined and determinable rules) — and we don’t have such a coherence in the CoqIDE source code.